docs/user/control-api.md

```diff
@@ -0,0 +1,189 @@
+# Controlling Glean
```

Comment:
Note: this is not the "control API", this is the "general API" (in contrast with the "specific metric API")

Reply:
And here I thought I could slip that in.